app/src/main/java/com/github/se/cyrcle/ui/parkingDetails/ParkingDetailsScreen.kt

```diff
@@ -330,14 +338,17 @@ fun ParkingDetailsScreen(
                       // There are images to display
                     } else {
                       LazyRow(
-                          modifier = Modifier.weight(2f).testTag("ParkingImagesRow"),
+                          modifier =
+                              Modifier.weight(2f).fillMaxHeight().testTag("ParkingImagesRow"),
                           horizontalArrangement = Arrangement.spacedBy(8.dp)) {
                             items(imagesUrls) { url ->
                               Image(
                                   painter = rememberAsyncImagePainter(url),
                                   contentDescription = "Parking Image",
-                                  modifier = Modifier.size(100.dp),
-                              )
+                                  modifier =
+                                      Modifier.aspectRatio(1.0F).fillMaxHeight().clickable {
```

Comment:
Sure, let me try

Reply:
I like it, it crop the image so that it always fit the size given, I needed to specify then that the maxwidth is 150.dp so that we get square images : 

Before/After
<img src="https://github.com/user-attachments/assets/f6e4c591-9fc6-460c-967e-2e002fd9ebd0" width=100><img src="https://github.com/user-attachments/assets/6dcd26a3-4945-4371-a177-be909beeaccf" width=100>